docs/llb/llb-local-image-upload.md

```diff
@@ -0,0 +1,24 @@
+# LLB Local Image Upload Guide
+This is a list of the steps required to locally upload images to LLB's S3.
+
+##Run Configurations
+1. Open the LLB backend project in IntelliJ
+2. Create a new Run Configuration for the ServiceMain class
+3. Add the environment variables AWS_ACCESS_KEY and AWS_SECRET_ACCESS_KEY with their corresponding values (ask Jack how they should do this)
```

Comment:
I think we should just share the keys we made via DM at this step, they're all for a public bucket that isn't used on the C4C account, so everyone should be able to use them without conflicts.